common/ux-window.cpp

```diff
@@ -21,6 +21,11 @@
 
 #include <iostream>
 
+void glfw_error_callback(int error, const char* description)
+{
+    std::cerr << "GLFW Driver Error: " << description << "\n";
```

Comment:
What about 'error'? No additional information needed?

Reply:
This is from GLFW reference code, I assume its not adding extra information